lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -319,34 +317,65 @@ def set_aspect(self, aspect, adjustable=None, anchor=None, share=False):
         """
         _api.check_in_list(('auto', 'equal', 'equalxy', 'equalyz', 'equalxz'),
                            aspect=aspect)
+        if adjustable is None:
+            adjustable = self._adjustable
+        _api.check_in_list(('box', 'datalim'), adjustable=adjustable)
         super().set_aspect(
             aspect='auto', adjustable=adjustable, anchor=anchor, share=share)
```

Comment:
If I've traced the code correctly, `super().set_aspect` calls `set_adjustable`, which calls `_api.check_in_list` for `adjustable`. Can it then be removed from here (since it's just code repetition)?

Reply:
Wondering if I have read this correctly. Removing this should also get rid of the code coverage warning (as will removing the `adjustable` parameter from the new test, but explicitly specifying it seems better).

Reply:
To get a code coverage warning on that line is quite unexpected since is it not conditionally executed...